indico/htdocs/js/indico/modules/category_management.js

```diff
@@ -83,4 +83,10 @@
             });
         }
     };
+
+    global.setupCategoryEventsList = function setupCategoryEventsList() {
+        $('.sortable-column').on('click', function() {
```

Comment:
I'd just wrap the title in an `<a>` tag. Personally I absolutely despise websites that use this kind of JS which prevents you from opening the thing in another tab.
You can use CSS to make the title `display:block` so you can click over the whole width, but I don't think this is necessary - people will most likely click the title anyway.